.github/workflows/docker_image.yml

```
@@ -0,0 +1,25 @@
+name: Build Docker Image
+
+on:
+  pull_request:
+    branches:
+      - main
```

Let's do this for consistency with the other files.
```suggestion
    branches: [ main ]
```